<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCollege</class>
 <widget class="QWidget" name="FormCollege">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#FormCollege
{
	
	background-color: #ffffff;
}
#widgetTop
{
	
	background-color: #FAFAFA;
}

QToolButton {
border: none;
border-radius: 3px;
background-color: #FAFAFA;
min-width: 80px;
font-family: &quot;Microsoft YaHei&quot;;
font-size:9pt;
font-weight: bold;
color:black;
}

QToolButton:hover {
border: none;
border-radius: 3px;
background-color: #EBF8FF;
min-width: 80px;
font-family: &quot;Microsoft YaHei&quot;;
font-size:9pt;
font-weight: bold;
color:gray;
}


QTableView
{
	border:none;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetTop" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="4">
       <widget class="QToolButton" name="tBtnRefresh">
        <property name="text">
         <string>  刷新</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/res/icon/zrefresh.png</normaloff>:/res/icon/zrefresh.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="tBtnDelete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>  删除</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/res/icon/zdelete.png</normaloff>:/res/icon/zdelete.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="tBtnEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>  编辑</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/res/icon/zedit.png</normaloff>:/res/icon/zedit.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="tBtnAdd">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>  添加</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/res/icon/zadd.png</normaloff>:/res/icon/zadd.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tableView"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
</ui>
